gap/precompiled_categories/FinQuiversAsCCCPrecompiled.gi

```diff
@@ -811,7 +811,7 @@ end
     if IsBound( cat!.precompiled_functions_added ) then
         
         # COVERAGE_IGNORE_NEXT_LINE
-        #Error( "precompiled functions have already been added before" );
+        Error( "precompiled functions have already been added before" );
```

Comment:
If you simply remove this change, the CI should pass.

Reply:
Thanks, done.